sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskEditForm.java

```diff
@@ -331,9 +331,12 @@ private void updateByCreatingAndAssignee() {
 			boolean creating = value.getCreationDate() == null;
 
 			UserDto user = UserProvider.getCurrent().getUser();
-			boolean creator = user.equals(value.getCreatorUser());
+			boolean creator = value.getCreatorUser() == null
+				? UserRole.getJurisdictionLevel(user.getUserRoles()) == JurisdictionLevel.NATION
+				: user.equals(value.getCreatorUser());
 			boolean supervisor = UserRole.isSupervisor(user.getUserRoles());
 			boolean assignee = user.equals(getFieldGroup().getField(TaskDto.ASSIGNEE_USER).getValue());
+			boolean observer = user.equals(getFieldGroup().getField(TaskDto.OBSERVER_USERS).getValue());
```

Comment:
That's not what I meant :) Remove this line, and instead add the `TaskDto.OBSERVER_USERS` field to the list of fields in the third setReadOnly call below (where you now added the `observer` boolean check)